test/movement.unittests.js

```diff
@@ -153,30 +153,30 @@ describe('Movement Helpers Unit Tests', function() {
         });
     });
 
-    describe('getPotentialEnemyCastleLocation Returns Values Correctly', function(done) {
-        it('getPotentialEnemyCastleLocation Returns Valid Value Given Valid Castle location And fullMap Objects With Valid Values', function(done) {
+    describe('getAttackerPatrolRoute Returns Values Correctly', function(done) {
+        it('getAttackerPatrolRoute Returns Valid Value Given Valid Castle location And fullMap Objects With Valid Values', function(done) {
```

Comment:
This change is because the getPotentialEnemyCastleLocation is not existent now, correct?

Reply:
yes, changed to getAttackerPatrolRoute, which will probably get modified a bit with the function to check map reflection